pywren_ibm_cloud/libs/docker/clients.py

```diff
@@ -0,0 +1,108 @@
+import os
```

Comment:
@omerb01 it should be one client per VM type... not all clients inside this one.